source/blog/2017-07-17-fedora-atomic-26-kickstarts.md

```diff
@@ -19,13 +19,21 @@ ostreesetup --osname="fedora-atomic" --remote="fedora-atomic-26" --url="file:///
 
 Three things changed here.  First, the `remote` is now &quot;fedora-atomic-26&quot; instead of just &quot;fedora-atomic&quot; (`osname` can really be whatever you want). This is a temporary measure until we [merge all of the repos for continuous upgrade](https://fedoramagazine.org/upcoming-fedora-atomic-lifecycle-changes/).  Second, the `url` to install from local media has changed to the simpler &quot;file:///ostree/repo&quot;.  Third, the upstream `ref` is now &quot;fedora/26/x86_64/atomic-host&quot;, which both removes the reference to a specific container runtime from the edition and makes the urls consistent with other Fedora Editions.
 
+We can also stop disabling cloud-init for Anaconda (bare metal) installs:
+
+```
+services --enabled="systemd-timesyncd"
+```
+
+Previously, we'd have added `--disabled="cloud-init,cloud-config,cloud-final,cloud-init-local"` in that line, but Anaconda now automatically disables cloud-init since it won't be used for a local install, so we don't have to.  
```

Comment:
clarification: anaconda doesn't disable it. We changed it to not be enabled by default and to only enable it in our cloudimage kickstarts. A more correct like would be *"but it is now not enabled by default and only enabled in our cloud image kickstarts, so we don't have to"*